test/verify/check-metrics

```diff
@@ -839,65 +839,107 @@ podman save quay.io/libpod/busybox | sudo -i -u admin podman load
         b.wait_not_present("#current-metrics-card-cpu .temperature")
 
         # No matching type
-        m.execute("mkdir -p /tmp/hwmon/hwmon0")
-        m.execute("echo BAT0 > /tmp/hwmon/hwmon0/name")
-        m.execute("mount -o bind /tmp /sys/class")
+        m.execute("mkdir -p /tmp/sensor-sys-class/hwmon/hwmon0")
+        self.addCleanup(m.execute, "rm -rf /tmp/sensor-sys-class")
+        m.write("/tmp/sensor-sys-class/hwmon/hwmon0/name", "BAT0")
+        m.write("/tmp/sensor-sys-class/hwmon/hwmon0/temp1_input", "40000")
+        m.execute("mount -o bind /tmp/sensor-sys-class /sys/class")
+        self.addCleanup(m.execute, "umount /sys/class")
+        self.login_and_go("/metrics")
 
         b.wait_not_present("#current-metrics-card-cpu .temperature")
 
         # create files that contain CPU temperature
-        m.execute("mkdir -p /tmp/hwmon/hwmon1")
+        # ARM
+        m.execute("mkdir -p /tmp/sensor-sys-class/hwmon/hwmon1")
+        m.write("/tmp/sensor-sys-class/hwmon/hwmon1/name", "cpu_thermal")
+        m.write("/tmp/sensor-sys-class/hwmon/hwmon1/temp1_input", "30000")
 
-        # Testing on ARM devices
-        m.execute("echo cpu_thermal > /tmp/hwmon/hwmon1/name")
-        m.execute("echo 60000 > /tmp/hwmon/hwmon1/temp1_input")
+        self.login_and_go("/metrics")
```

Comment:
Curious that this works without a previous `b.logout()`. I don't see anything in `login_and_go()` which would log you out first.